ui/src/app/machines/views/MachineDetails/MachineSummary/OverviewCard/StatusCard/StatusCard.tsx

```diff
@@ -81,6 +94,40 @@ const StatusCard = ({ machine }: Props): JSX.Element => {
             </Tooltip>
           </p>
         ) : null}
+        {machine.status === NodeStatus.DEPLOYED && machine.enable_hw_sync ? (
+          <>
+            <hr />
+            <p className="u-text--muted">
+              Periodic hardware sync enabled{" "}
+              {/* TODO: Update docs links https://github.com/canonical-web-and-design/app-tribe/issues/787 */}
+              {/* TODO: use actual `sync_interval` value from the back-end https://github.com/canonical-web-and-design/app-tribe/issues/782 */}
+              <Tooltip
+                position="right"
+                message={
+                  <>
+                    This machine hardware info is synced every{" "}
+                    {formatSyncInterval(machine.sync_interval)}.{"\n"}
+                    You can check it at the bottom, in the status bar.{"\n"}More
+                    about this in the{" "}
+                    <a className="p-link--inverted" href="#">
```

Comment:
This also is (as mentioned in the other PR) causing issues with the console output due to the lint errors.

Reply:
Thanks! Updated.